app/main.py

```diff
@@ -14,4 +14,11 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    count = 0
+
+    for ch in phrase:
```

Comment:
```suggestion
    for char in phrase:
```
use complete words for variable names